pick.c

```diff
@@ -11,6 +11,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <sys/ioctl.h>
```

Comment:
Should be placed on a separate line before the other includes, see [style(9)](http://man.openbsd.org/style).